Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/edit.go

```diff
@@ -294,14 +295,15 @@ outter:
 			}
 
 			if hold, msg := jsonmerge.TestPreconditionsHold(patch, preconditions); !hold {
-				fmt.Fprintf(out, "error: %s", msg)
+				fmt.Fprintf(out, "Error: %s\n", msg)
```

Comment:
We use `error:` in pkg/kubectl/cmd/util#HandleErrors, so we should use `error:` in edit.